pallets/stateful-storage/README.md

```diff
@@ -0,0 +1,60 @@
+# Stateful Storage Pallet
+
+The Stateful Storage pallet provides per-MSA functionality for reading and writing stateful data (where the relevant piece is solely the latest state).
+
+## Summary
+
+When a Schema uses `Paginated` or `Itemized`, the payload data is stored in this pallet.
+Data stored is user-centric instead of time-centric like the Messages Pallet.
+The pallet storage use [child storage](https://paritytech.github.io/polkadot-sdk/master/frame_support/storage/child/index.html), so RPC access to data is suggested over queries.
```

Comment:
Should read "The pallet storage uses..."

Reply:
Also "so" in this statement is unclear.

Reply:
Revised.